swagger-spec/swagger.yaml

```diff
@@ -294,3 +294,13 @@ paths:
 
   /wikis/{wiki_id}/content/:
     $ref: 'wikis/content.yaml'
+
+
+  ###################
+  #   METASCHEMAS   #
```

Comment:
Can you make sure that this file stays in alphabetical order? The metaschemas URLs should be between institutions and nodes here.